tracker/templates/challenges.json.j2

```diff
@@ -0,0 +1,49 @@
+    {
+      "name": "append-no-conflicts",
+      "description": "Indexes the whole document corpus using Elasticsearch default settings. We only adjust the number of replicas as we benchmark a single node cluster and Rally will only start the benchmark if the cluster turns green. Document ids are unique so all index operations are append only. After that a couple of queries are run.",
+      "default": true,
+      "schedule": [
+        {
+          "operation": "delete-index"
+        },
+        {
+          "operation": {
+            "operation-type": "create-index",
+            "settings": {% raw %}{{index_settings | default({}) | tojson}}{% endraw %}
+          }
+        },
+        {
+          "operation": "index-append",
+          "warmup-time-period": 120,
+          "clients": {% raw %}{{bulk_indexing_clients | default(8)}}{% endraw %}
+        },
```

Comment:
IMHO it's ok to drop every task below this line for the initial track template.

Reply:
https://github.com/elastic/rally/commit/1ef7f2addd8fae5833c82cdf502b317594aa6d80